src/controller/CommissioneeDeviceProxy.cpp

```diff
@@ -132,6 +132,13 @@ CHIP_ERROR CommissioneeDeviceProxy::UpdateDeviceData(const Transport::PeerAddres
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR CommissioneeDeviceProxy::SetConnected()
+{
+    mState = ConnectionState::SecureConnected;
+    bool didLoad;
+    return LoadSecureSessionParametersIfNeeded(didLoad);
```

Comment:
Any reason why we ignore the value? If we ignore it, perhaps rename to `didLoadPlaceholder` ?

Reply:
`_` is sometimes used for unused values (like `_didLoad`) if you find that shorter. 
Would not block the PR on this though - it seems reasonably readable that when becoming connection, we ensure we have parameters but do not care if they were loaded or already available. It is a side-effect of having an 'if' in the method name.

Reply:
changed to _didLoad